refgenconf/const.py

```diff
@@ -15,22 +15,25 @@
 CFG_ARCHIVE_KEY = "genome_archive"
 CFG_GENOMES_KEY = "genomes"
 
+CFG_GENOME_DESC = "genome_description"
```

Comment:
Cool, this answers a question I had while reading the paper about why not just use "description!"